android/build.gradle

```diff
@@ -34,6 +34,12 @@ repositories {
 dependencies {
     implementation 'com.braintreepayments.api:drop-in:4.+'
     implementation 'com.facebook.react:react-native:+'
+    // Braintree dependencies
```

Comment:
It would be better for users to make this a configuration step for using the library
Not everybody wants to have this in their code
See example of 3D Secure https://github.com/bamlab/react-native-braintree-payments-drop-in#3d-secure

Reply:
These library must be in library's gradle. If not, there will be an error like below
![Screenshot at Nov 11 09-19-50](https://user-images.githubusercontent.com/16470443/68556347-84d4ca00-0464-11ea-8c2d-cf03960be515.png)